src/popup/components/tool.jsx

```diff
@@ -10,13 +11,27 @@ import NoTickets from './no-tickets';
 import TicketControls from './ticket-controls';
 
 function Tool({ tickets, errors }) {
+  const { close, openopts } = useContext(EnvContext);
+
+  async function onClickSettingsLink(event) {
+    event.preventDefault();
+    await openopts();
+    close();
+  }
+
   return (
     <>
       <Navbar>
         <span className="navbar-brand">Tickety-Tick</span>
         <ul className="navbar-nav ml-auto">
           <li className="nav-item">
-            <Link className="nav-link" to="/about">
+            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+            <a className="nav-link" href="#" onClick={onClickSettingsLink}>
```

Comment:
Unfortunately the `.nav-link` style does not properly style buttons. 😞 I didn't want to go down a 🐇🕳️ to try and make them work. I'll have another look though whether using `.btn-link` in combination with `.nav-link` can trick a button into looking good there. ✌️

Reply:
Tried with making it a button and applying the `btn` and `btn-link` classes, but that still makes them look and behave different from the help link then.

Instead, I opted for updating the `href` attribute to point to `options.html`. This is actually valid and if you didn't have the `onClick` handler, it would look like this: 😀 

![options page in popup](https://user-images.githubusercontent.com/706519/101471142-db9a2500-393e-11eb-87c4-a1db547bf16a.png)

Reply:
Haha, would have expected this to work 😂